test/e2e/proxy.go

```diff
@@ -52,6 +52,7 @@ func proxyContext(version string) {
 	f := NewFramework("proxy")
 	prefix := "/api/" + version
 
+	// Port here has to be kept in sync with default kubelet port.
```

Comment:
there is a const for this:

https://github.com/kubernetes/kubernetes/blob/master/pkg/master/ports/ports.go#L28